app/src/main/java/org/jellyfin/androidtv/ui/shared/OutlineSpan.kt

```diff
@@ -0,0 +1,54 @@
+package org.jellyfin.androidtv.ui.shared
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.text.style.ReplacementSpan
+
+/**
+ * A class that draws the outlines of a text
+ */
+class OutlineSpan(
+): ReplacementSpan() {
+
+	override fun getSize(
+		paint: Paint,
+		text: CharSequence,
+		start: Int,
+		end: Int,
+		fm: Paint.FontMetricsInt?
+	): Int {
+		if (fm != null) {
+			fm.ascent = paint.fontMetricsInt.ascent
+			fm.bottom = paint.fontMetricsInt.bottom
+			fm.descent = paint.fontMetricsInt.descent
+			fm.leading = paint.fontMetricsInt.leading
+			fm.top = paint.fontMetricsInt.top
+		}
+
+		return paint.measureText(text, start, end).toInt()
+	}
```

Comment:
Not 100% sure about this but I think this function does nothing right now, some values in `fm` get changes but `fm` is never used after that.

Reply:
Without that when two line subtitles are shown they the first line is half cut and displaying over the other line.